glue/app/glue_application.py

```diff
@@ -15,21 +16,23 @@
 from ..external.qt.QtCore import Qt, QSize, QSettings, Signal
 
 from ..core import command
-from .. import env
-from ..qt import get_qapp
-from .decorators import set_cursor, messagebox_on_error
 from ..core.application_base import Application
 
-from .actions import act
-from .qtutil import (pick_class, data_wizard,
-                     GlueTabBar, load_ui, get_icon, nonpartial)
-from .widgets.glue_mdi_area import GlueMdiArea, GlueMdiSubWindow
-from .widgets.edit_subset_mode_toolbar import EditSubsetModeToolBar
-from .widgets.layer_tree_widget import PlotAction, LayerTreeWidget
-from .widgets.data_viewer import DataViewer
-from .widgets.settings_editor import SettingsEditor
-from .widgets.mpl_widget import defer_draw
-from .feedback import submit_bug_report
+from .. import env
+from ..qt import get_qapp
+from ..qt.decorators import set_cursor, messagebox_on_error
+from ..qt.actions import act
+from ..qt.qtutil import (pick_class, data_wizard,
+                         GlueTabBar, load_ui, get_icon, nonpartial)
+from ..qt.widgets.glue_mdi_area import GlueMdiArea, GlueMdiSubWindow
+from ..qt.widgets.edit_subset_mode_toolbar import EditSubsetModeToolBar
+from ..qt.widgets.layer_tree_widget import PlotAction, LayerTreeWidget
+from ..qt.widgets.data_viewer import DataViewer
+from ..qt.widgets.settings_editor import SettingsEditor
+from ..qt.widgets.mpl_widget import defer_draw
+from ..qt.feedback import submit_bug_report
+
+ROOT = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I never get this logic exactly right -- have you checked that if it works when used:

- As a mac app
- From python setup.py develop
- From pip install glue-viz